### ISSUE TITLE
Park name there at zoom 15, gone at 16, back at 17

### ISSUE THREAD

**Issue by jidanni** [gravitystorm/openstreetmap-carto#4856]
The location is [GPS 42.209706 N 87.833715 W] https://www.openstreetmap.org/?mlat=42.2097057&mlon=-87.8337153#map=16/42.2097057/-87.8337153 . I can't find the duplicate bug today.

**Comment by dch0ph** [comment]
I fear this is a feature of the current style rather than a bug. It is trying to place the park name at a single location in the "middle" of the area where other stuff is happening. The park name is probably colliding or not colliding with the "Trail Way" road name depending on where the "Trail Way" label is placed along the road (which will vary with zoom level).

These collisions are more likely since the landuse labels (such as the park name) grow at the feature becomes bigger, but they are rendered, together with simple amenity points (text-point layer) after road names (roads-text-name).

There's an argument for separating landuse-type labelling from simple amenity-type labelling and moving it further up the layer stack, and before road labels. Way labels can fit around point-type labels, while point-type labels are all or nothing.

Mapnik does have the ability to try different sizes to find one that fits, but I don't think CartoCSS supports this?

**Comment by imagico** [comment]
Again: Duplicate of #3880.

@jidanni - if, just a few weeks after #4876 and #4859, you cannot remember that you have reported symptoms of this over and over again, then you might need to consider either developing a reliable way to remember this or refraining from opening new issues.

**Comment by pnorman** [comment]
@jidanni, counting this issue you have reported *six* (#4831, #4852, #4859, #4876, #4828, and this one) duplicates of #3880. These duplicate issues add no value and you have twice (https://github.com/gravitystorm/openstreetmap-carto/issues/4876#issuecomment-1737157121, https://github.com/gravitystorm/openstreetmap-carto/issues/4859#issuecomment-1676136257) been told to stop.

Do not open any more issues that are a duplicate of #3880. It does not matter if you can find the issue or not, because you know there is one. Additionally, more examples of that issue bring no value, so don't post them in #3880.

**Comment by jidanni** [comment]
I am sorry. I just try to report weird things that I see. For instance today I saw at https://www.openstreetmap.org/#map=17/37.86520/-100.56460 this weird stuff that reminded me of the Jimi Hendrix song, "If 6 Was 9."

![Screenshot_20231014-085046.jpg](https://github.com/gravitystorm/openstreetmap-carto/assets/1230959/4560cf42-9ff0-422a-af2f-a73d088cfe52)

![Screenshot_20231014-085028.jpg](https://github.com/gravitystorm/openstreetmap-carto/assets/1230959/8f93c20f-499b-4725-bdd8-d2c7dd89f031)

![Screenshot_20231014-085005.jpg](https://github.com/gravitystorm/openstreetmap-carto/assets/1230959/bb6349f9-70b1-4715-889d-baf17f3b9906)

![Screenshot_20231014-084946.jpg](https://github.com/gravitystorm/openstreetmap-carto/assets/1230959/db7f5e9c-7641-418c-ad7c-5bb872c6f654)

![Screenshot_20231014-084922.jpg](https://github.com/gravitystorm/openstreetmap-carto/assets/1230959/a0870d51-aa1f-4aeb-a1dc-300603825013)

I don't know if it's a new bug, or an existing bug. Think of me as that confused person calling the 800 number trying to report a problem that they see. And then the customer service representative hands it off to the higher layers to figure out what's going on.

I mean it's great that some people can see problems and other people can fix them and then then there's people that can see them and figure out what they are at the same time etc but not all these people can do all these things, some can only do one.

Anyway I don't know if I found something new with the 6 9 thing or not here above. So like you say, I dare not create a new bug report. Thanks.

**Comment by jidanni** [comment]
Probably above, there is the problem with almost perfectly north south roads that words could easily flip. But they have to flip somewhere within 180° so that is understandable.

Then there maybe is another problem of after they're flipped they become slightly longer so their abbreviation is only used instead of their full name.

So there is an unfortunate incidence of multiple things happening to cause the above optical illusion.

Anyway, so maybe they're new problems, or maybe they're a combination of old problems. I could just not report any of them ever and then maybe you'd lose a couple bugs. Or I could report all of them but then you gain many many duplicate bugs. So I'll just append it to this bug because I really can't help much

**Comment by imagico** [comment]
> I don't know if it's a new bug, or an existing bug. Think of me as that confused person calling the 800 number trying to report a problem that they see. And then the customer service representative hands it off to the higher layers to figure out what's going on.

And if this was a commercial project with customer support staff on call looking through and assessing/categorizing every report of weirdness this expectation would make sense because paying customers can expect the company they pay to provide this kind of service.

But that is not the case here.  This is a cooperative community project.  Typically, no one gets paid for work on this project (and in the rare exceptions when this is the case it is not for addressing reports from map users).  And cooperation means that working _together_ for improving the style stands in the foreground.  Every contributor tries to provide something of use and benefit to the project and tries to adjust the way they do that to the needs of the project and of the others whose cooperation they depend on for their contributions to be useful.

If you just want to collect and publicly document weird observations in the style but are not interested in analyzing these observations and attributing them to existing issues on this tracker - even in cases where this is fairly simple and you have been advised of the existing issue about that in the past (symbol/label vanishing and reappearing as you zoom in/out -> #3880) then the better approach might be to document these observations externally on a service of your choosing.

Regarding the observation about _6 and 9_ - that is in both aspects it represents (orientation of line labels problem and styling similarity between name and ref labels on residential roads) a so far unreported problem AFAIK (the latter was already discussed in #3709 though).  Anyone is welcome to open an issue on either of these.

**Comment by jidanni** [comment]
Okay I created a new report bug for it.

As far as users who just want to report the weird things they see, perhaps on https://community.openstreetmap.org/ two new sections could be added,
1. Odd map rendering
2. Odd map items

Then others can discuss them and if a real bug is found, they could be elevated to GitHub.
